### PR TITLE
refactor(prejoin) use jss instead of sass in CallingDialog

### PR DESCRIPTION
**Pull request by c0m1t**
Convert Sass to JSS in `CallingDialog` component.

### CODE REVIEW

File: react/features/prejoin/components/dialogs/CallingDialog.js [hunk 1]
```diff
@@ -35,6 +37,30 @@ type Props = {
     t: Function,
 };
 
+const useStyles = makeStyles(theme => {
+    return {
+        root: {
```
Comment (robertpin) [inline react/features/prejoin/components/dialogs/CallingDialog.js:42]: Rename this to `prejoinDialogCalling`?

File: react/features/prejoin/components/dialogs/CallingDialog.js [hunk 2]
```diff
@@ -35,6 +37,30 @@ type Props = {
...
+            padding: theme.spacing(3),
+            textAlign: 'center',
+
+            '& .prejoin-dialog-calling-header': {
+                textAlign: 'right'
+            },
+
+            '& .prejoin-dialog-calling-label': {
+                fontSize: '15px',
+                margin: `${theme.spacing(2)} 0 ${theme.spacing(2)} 0`
```
Comment (robertpin) [inline react/features/prejoin/components/dialogs/CallingDialog.js:52]: This seems to be different from how it was in SCSS. Is it intentional?

File: react/features/prejoin/components/dialogs/CallingDialog.js [hunk 3]
```diff
@@ -43,9 +69,10 @@ type Props = {
  */
 function CallingDialog(props: Props) {
     const { number, onClose, status, t } = props;
+    const classes = useStyles();
 
     return (
-        <div className = 'prejoin-dialog-calling'>
+        <div className = { clsx('prejoin-dialog-calling', classes.root) }>
```
Comment (robertpin) [inline react/features/prejoin/components/dialogs/CallingDialog.js:75]: Please remove the old class

**Review by robertpin — CHANGES_REQUESTED** [review]

File: react/features/prejoin/components/dialogs/CallingDialog.js (hunk 2 above)
Comment (c0m1t) [reply react/features/prejoin/components/dialogs/CallingDialog.js:52]: No It was a typo. Thank you.

**Review by robertpin — APPROVED** [review]

**Comment by robertpin** [comment]
@c0m1t there's a lint issue on this one

**Review by robertpin — APPROVED** [review]

**Pull request merged**